tests/test_default_sprites.py

```diff
@@ -42,3 +42,117 @@ def test_target_sprite_min_speed():
     target_sprite.on_update(ppb.events.Update(1.0), lambda x: None)
 
     assert target_sprite.position.isclose((-1.2, -1.6))
+
+
+class TestWASDDirectionKeys:
+    def test_up(self):
+        WASDDirectionKeys().up == ppb.keycodes.W
```

Comment:
I just noticed, these tests don't seem to do anything?

Reply:
yea sorry was gonna remove them